cli/flox-rust-sdk/src/models/environment/remote_environment.rs

```diff
@@ -48,6 +48,9 @@ pub enum RemoteEnvironmentError {
 
     #[error("could not create temporary environment")]
     CreateTempDotFlox(#[source] std::io::Error),
+
+    #[error("could not update out link ({1})")]
```

Comment:
Suggestion: if this message could end up user-facing it probably needs to have a description that would be meaningful to a user.

Reply:
We have conversions into user friendly errors in the errors module, I didnt write one for this variant (i just pass this through) because i think this error woule rather be a bug than anything a user needs to deal with. I guess we need some sort of tag somewhere what is but and what not but wither way these errors are more dev/us focused IMO.